mk/spksrc.download.mk

```diff
@@ -128,7 +128,7 @@ download_target: $(PRE_DOWNLOAD_TARGET)
 	        rm -f $${localFile}.part ; \
 	        url=`echo $${url} | sed -e '#^\(http://sourceforge\.net/.*\)$#\1?use_mirror=autoselect#'` ; \
 	        echo "wget $${url}" ; \
-	        wget $(DOWNLOAD_ARGS) -nv -O $${localFile}.part $${url} ; \
+	        wget --secure-protocol=TLSv1_2 -nv -O $${localFile}.part $${url} ; \
```

Comment:
I wonder if it may be better to replace wget by curl here

Reply:
I tried to use curl, but curl fails to download from ftp:// sites (wget has no problems with ftp://)